scripts/publish-vsix.js

```diff
@@ -8,17 +8,24 @@ const packageVersion = JSON.parse(shell.cat('package.json')).version;
 const vsix = shell.ls().filter(file => file.match(`-${packageVersion}.vsix`));
 
 if (!vsix.length) {
-  shell.error('No VSIX found matching the requested version in package.json');
+  console.log('No VSIX found matching the requested version in package.json');
```

Comment:
Changed this to console.log since shell.error does not surface the error message.